docs/release-notes/highlights-2.15.0.asciidoc

```diff
@@ -0,0 +1,21 @@
+[[release-highlights-2.15.0]]
+== 2.15.0 release highlights
+
+[float]
+[id="{p}-2150-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 2.15.0 of {n}. Check <<release-notes-2.15.0>> for the full list of changes.
+
+[float]
+[id="{p}-2150-wolfi-images"]
+=== Wolfi images
+
+ECK has transitioned to using https://github.com/wolfi-dev[Wolfi images], an open-source project aimed at delivering a secure, minimal base image for containerized applications. By adopting Wolfi, ECK strengthens its security posture, significantly lowering the potential for CVEs (Common Vulnerabilities and Exposures) and reducing the overall attack surface. This shift aligns with best practices for container security, providing a more robust foundation for running workloads reliably and securely.
+
+[float]
+[id="{p}-2150-latest-k8s-support"]
+=== Support for Kubernetes 1.31
```

Comment:
I’m not sure if it’s worth mentioning here, since we didn’t make any changes specific to the latest K8s release.

Reply:
Yes, we didn't usually mention this.

Reply:
+1 for leaving this out to stay consistent with previous messaging.

Reply:
Thanks for the feedback. I'll remove this part from the highlights.

If we simplify the Wolfi adoption section, maybe we could position this as a ‘maintenance’ release focused on enhancements, documentation improvements, and dependency updates. This would make the release highlights shorter than usual, without calling out specific changes. What do you think, @pebrc, @barkbay?